src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -0,0 +1,368 @@
+<?php
```

Comment:
This file comes from https://github.com/WordPress/gutenberg/blob/trunk/lib/class-wp-theme-json-resolver.php but I've removed the bits that weren't necessary just yet (user data, etc).